sematic/ui/packages/main/src/pipelines/RunMetricsPanel.tsx

```diff
@@ -0,0 +1,107 @@
+import { Timeline } from "@mui/icons-material";
+import { Box, Typography, useTheme } from "@mui/material";
+import { useCallback, useEffect, useMemo, useState } from "react";
+import { Link } from "react-router-dom";
+import Loading from "src/components/Loading";
+import { TimeseriesMetric } from "src/components/Metrics";
+import { METRIC_SCOPES } from "@sematic/common/src/constants";
+import { MetricsFilter, useListMetrics } from "@sematic/common/src/hooks/metricsHooks";
+import { usePipelinePanelsContext } from "src/hooks/pipelineHooks";
+import { getChartColor } from "src/utils";
+import {
+    MetricPoint,
+    runIsInTerminalState,
+} from "@sematic/common/lib/src/Models";
+import { metricsSocket } from "src/sockets";
+
+export default function RunMetricsPanel() {
+    const theme = useTheme();
+
+    const { selectedRun } = usePipelinePanelsContext();
+
+    let run = selectedRun!;
+
+    const [latestBroadcastEvent, setLatestBroadcastEvent] = useState<
```

Comment:
I suggest we rename this to `metricPoints`, and rename the formal parameters of wherever it was passed (including `<TimeseriesMetric>`, `useMetrics()` and `useListMetrics()`).  I found it  hard to follow when it's called `event` when passed to `useMetrics()`